Source/Scene/getClippingFunction.js

```diff
@@ -0,0 +1,168 @@
+define([
+        '../Core/Check',
+        '../Renderer/PixelDatatype',
+        './getUnpackFloatFunction'
+    ], function(
+        Check,
+        PixelDatatype,
+        getUnpackFloatFunction) {
+    'use strict';
+
+    /**
+     * Gets the glsl functions needed to retrieve clipping planes from a ClippingPlaneCollection's texture.
+     *
+     * @param {ClippingPlaneCollection} clippingPlaneCollection ClippingPlaneCollection with a defined texture.
+     * @returns {String} A string containing glsl functions for retrieving clipping planes.
+     * @private
+     */
+    function getClippingFunction(clippingPlaneCollection) {
+        //>>includeStart('debug', pragmas.debug);
+        Check.typeOf.object('clippingPlaneCollection', clippingPlaneCollection);
+        //>>includeEnd('debug');
+        var unionClippingRegions = clippingPlaneCollection.unionClippingRegions;
+        var clippingPlanesLength = clippingPlaneCollection.length;
+        var texture = clippingPlaneCollection.texture;
+        var usingFloatTexture = texture.pixelDatatype === PixelDatatype.FLOAT;
+        var width = texture.width;
+        var height = texture.height;
+
+        var functions = usingFloatTexture ? getClippingPlaneFloat(width, height) : getClippingPlaneUint8(width, height);
+        functions += '\n';
+        functions += unionClippingRegions ? clippingFunctionUnion(usingFloatTexture, clippingPlanesLength) : clippingFunctionIntersect(usingFloatTexture, clippingPlanesLength);
+        return functions;
+    }
+
+    function clippingFunctionUnion(usingFloatTexture, clippingPlanesLength) {
+        var functionString =
+            'float clip(vec4 fragCoord, sampler2D clippingPlanes, mat4 clippingPlanesMatrix)\n' +
+            '{\n' +
+            '    vec4 position = czm_windowToEyeCoordinates(fragCoord);\n' +
+            '    vec3 clipNormal = vec3(0.0);\n' +
+            '    vec3 clipPosition = vec3(0.0);\n' +
+            '    float clipAmount = 0.0;\n' +
+            '    float pixelWidth = czm_metersPerPixel(position);\n' +
```

Comment:
@lilleyse dependence on this function causes edge styling to break in orthographic view.
Just setting it to a constant when we switch to orthographic seems fine, but that will need shader recompilation.
Separate PR later?

Reply:
Discussed offline, should be a quick fix.

Reply:
Nice fix!